8/community/run.sh

```diff
@@ -1,31 +1,38 @@
 #!/usr/bin/env bash
 
-set -e
+set -eou pipefail
 
-if [ "${1:0:1}" != '-' ]; then
-  exec "$@"
+if [[ "${1:-}" != -* ]]; then
+  exec "$@" 
 fi
 
+declare -a sq_opts
+
+add_env_var_as_env_prop() {
+  if [ ! -z "$1" ]; then
```

Comment:
I believe this is identical, shorter, and quite intuitive:
```suggestion
  if [ "$1" ]; then
```